### DISCUSSION TITLE
Is there a hot key for 'Convert measurement to a line string' when annotating?

### DISCUSSION THREAD

**Discussion by TimWOceanSciences** [biigle/core#926] [Q&A] [answered]
Hello

I'm creating a lot of lines with the line measurement tool and it would be very handy to a hot key for 'Convert measurement to a line string'. Does any one know of one or how I may set one up?

Thanks

Tim 

![Screenshot 2024-09-19 105221](https://github.com/user-attachments/assets/0d62b02d-9693-4a01-9cce-42c7ab4d40a8)

![image](https://github.com/user-attachments/assets/9d529a09-5f54-4f2c-aa5c-a9c5a7a174f1)

**Comment by mzur** [comment, marked as answer]
We had this as a feature idea for some time (https://github.com/biigle/core/issues/475) but it was not implemented yet.

**Reply by TimWOceanSciences** [reply]
OK thank you for the answer. A hot key for this would be very handy I think, if you ever get the chance. Cheers.

**Reply by TimWOceanSciences** [reply]
Although I just realised you can draw a line string on its own and just double click with the same result as going via the measuring tool. Sorry.
